context/brand-context/springernature/scss/40-base/typography.scss

```diff
@@ -0,0 +1,27 @@
+/*
+ * Headings
+ */
+
+h1 {
+	@include u-h1();
+}
+
+h2 {
+	@include u-h2();
+}
+
+h3 {
+	@include u-h3();
+}
+
+h4 {
+	@include u-h4();
+}
+
+h5 {
+	@include u-h4();
+}
```

Comment:
should this be `u-h5()`? the mixin exists

Reply:
It should!